scripts/deploy/310-openstack-services-extended.sh

```diff
@@ -11,6 +11,12 @@ osism wait --output --format script --delay 2 $task_ids
 task_ids=$(osism apply --no-wait --format script ceilometer2>&1)
 task_ids+=" "$(osism apply --no-wait --format script heat 2>&1)
 task_ids+=" "$(osism apply --no-wait --format script senlin 2>&1)
-task_ids+=" "$(osism apply --no-wait --format script skyline 2>&1)
+
+MANAGER_VERSION=$(docker inspect --format '{{ index .Config.Labels "org.opencontainers.image.version"}}' osism-ansible)
+if [[ $MANAGER_VERSION == "4.0.0" || $MANAGER_VERSION == "4.1.0" || $MANAGER_VERSION == "4.2.0" || $MANAGER_VERSION == "4.3.0" ]]; then
```

Comment:
```suggestion
if [[ $MANAGER_VERSION =~ ^4\.[0-9]\.[0-9]$ ]]; then
```
this would match all future 4.x.y releases as well, if we stick to single digit release numbers for minor and patch releases.